src/entities/session/model/store.ts

```diff
@@ -0,0 +1,26 @@
+import { createDomain, createEffect, sample } from "effector";
+import { login, logout, getAuthStatus, getCurrentUser } from "./api";
+import { User } from "./types";
+
+const domain = createDomain("entities/session");
+
+export const onLogin = domain.event();
```

Comment:
reade about naming in effector. Events must be in past. forexample - loginRequested